app/design/frontend/rwd/default/template/downloadable/catalog/product/type.phtml

```diff
@@ -39,5 +39,3 @@
     </p>
 <?php endif ?>
 <?php echo $this->getChildHtml('product_type_data_extra') ?>
-<?php //echo $this->getPriceHtml($_product) ?>
```

Comment:
See the base template: https://github.com/OpenMage/magento-lts/blob/ee46ff9b9a45809d5ebc7b09c13a91844f7c402c/app/design/frontend/base/default/template/downloadable/catalog/product/type.phtml#L36-L37

I would keep it for reference.

Reply:
restored